src/BlockExtractor.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace WP_CLI\I18n;
+
+use Gettext\Extractors\Extractor;
+use Gettext\Extractors\ExtractorInterface;
+use Gettext\Translations;
+use WP_CLI;
+
+final class BlockExtractor extends Extractor implements ExtractorInterface {
+	use IterableCodeExtractor;
+
+	/**
+	 * @inheritdoc
+	 */
+	public static function fromString( $string, Translations $translations, array $options = [] ) {
+		$file = $options['file'];
+		WP_CLI::debug( "Parsing file {$file}" );
```

Comment:
Debug statements should include a group as the second argument.

Reply:
We use `make-pot` as the group elsewhere already. Will re-use that onne.

Reply:
Done in 0b2e92bf8d1dbdc43286a8b6e81fc826b8355875